docs/USER_RECORD.md

```diff
@@ -234,6 +234,16 @@ optional, when unset the user should not be considered part of any realm. A
 user record with a realm set is never compatible (for the purpose of updates,
 see above) with a user record without one set, even if the `userName` field matches.
 
+`blobDirectory` → The absolute path to a world-readable copy of the user's blob
+directory. See [Blob Directories](USER_RECORD_BLOB_DIRS.md) for more details.
+
+`blobManifest` → An object, which maps blob directory filenames to
+SHA256 hashes. This exists for the purpose of including the contents
+of the blob directory in the record's signature. Managers that support
+blob directories and utilize signed user records (like `systemd-homed`)
+should use this field to verify the contents of the blob directory whenever
+appropriate.
```

Comment:
maybe say that the keys in this object are the blob names, and which characters are OK for it. And say the values should be sha256 hashes of the blob values, formatted in hex strings